src/index.html

```diff
@@ -11,12 +11,54 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+    />
+
+    <link
+      rel="stylesheet"
+      href="style.css"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#">
+        <img
+          src="images/logo.png"
+          alt="Logo"
+        />
+      </a>
+      <nav class="navigation">
+        <ul>
+          <li>
+            <a
+              href="#"
+              class="is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li><a href="#">SUMSUNG</a></li>
```

Comment:
Fix code style everywhere
```suggestion
          <li class="nav__item">
            <a href="#" class="nav__link">SAMSUNG</a>
          </li>
```